checker/build.xml

```diff
@@ -982,7 +982,7 @@
 
     <target name="-jtreg.download" depends="jtreg.check" unless="jtreg.exists">
       <get
-        src="https://adopt-openjdk.ci.cloudbees.com/view/OpenJDK/job/jtreg/lastSuccessfulBuild/artifact/jtreg-4.1-b12.tar.gz"
+        src="https://adopt-openjdk.ci.cloudbees.com/job/jtreg/lastSuccessfulBuild/artifact/jtreg-4.2-b02.tar.gz"
```

Comment:
What do you think of using `https://adopt-openjdk.ci.cloudbees.com/job/jtreg/lastSuccessfulBuild/artifact/jtreg-4.2.0-tip.tar.gz` to automatically get updates within 4.2.0 automatically? jtreg in general seems quite stable.